lwt/server/Http.re

```diff
@@ -28,10 +57,15 @@ let make_request_handler: Server.t('s, 'r, 'a, 'b) => request_handler =
       let res = Httpaf.Response.create(status, ~headers);
       Httpaf.Reqd.respond_with_string(reqd, res, content);
     };
-    server
-    |> Server.middleware
-    |> Server.Middleware.run(closer, respond, req)
-    |> ignore;
+    read_body()
+    >|= (
```

Comment:
This is where I was struggling, as I was using the bind operator `>>=` but I couldn't manage to get the proper return type.

By looking at the MCU example, I ended up changing the signature of `request_handler` to return a `Lwt.t(unit)`. Please let me know if this is ok or if there is another way to do that.

Reply:
We can ignore this return value 🙌 

```reason
read_body()
>|= (body_string => {
 /* ... */
})
|> ignore;
````

Reply:
🤦‍♂️

Of course, I got stuck in the 🐇 hole 🙃

Here we go: https://github.com/ostera/httpkit/pull/13/commits/9f15e8dc3ff8a594663eba456e372074c6b67c2c